http/file_server.ts

```diff
@@ -24,6 +24,13 @@ interface EntryInfo {
   name: string;
 }
 
+const envPermissionStatus = await Deno.permissions.query({
```

Comment:
Top-level-await often causes unexpected behavior to dependents. I'd suggest we should use querySync instead. (And querySync is now unavailable in Deploy. I think we should do something like the below for now: https://github.com/denoland/deno_std/pull/3178/files#diff-a65e7827e30b4089194fd08ed758f7344da9052d00a6d45c41ebae713c959522R37-R39

Reply:
Makes sense... updated!